photutils/psf/groupstars.py

```diff
@@ -0,0 +1,148 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Module which provides classes to perform source grouping."""
+
+from __future__ import division
+import abc
+import numpy as np
+from astropy.table import Column, Table, vstack
+
+
+__all__ = ['DAOGroup', 'GroupStarsBase']
+
+
+class GroupStarsBase(object):
+    __metaclass__ = abc.ABCMeta
```

Comment:
And if you want the class methods to inherit the documentation from the base class use:

```
from astropy.utils.misc import InheritDocstrings
class _ABCMetaAndInheritDocstrings(InheritDocstrings, abc.ABCMeta):
    pass


@six.add_metaclass(_ABCMetaAndInheritDocstrings)
class GroupStarsBase(object):
    ...
```